lib/rubocop/cop/rspec/each.rb

```diff
@@ -0,0 +1,53 @@
+module RuboCop
+  module Cop
+    module RSpec
+      # Check that all matcher is used instead of iterating over an array.
+      #
+      # @example
+      #   # bad
+      #   it 'validates users' do
+      #     [user1, user2, user3].each { |user| expect(user).to be_valid }
+      #   end
+      #
+      #   # good
+      #   it 'validates users' do
+      #     expect([user1, user2, user3]).to all(be_valid)
+      #   end
+      class Each < Cop
```

Comment:
I think `IteratedExpectation` is actually a pretty reasonable name. Since we may possibly have cops in the future relating to the `all` matcher, `AllMatcher` may be too general also.